docs/feature/policies/overview.md

```diff
@@ -272,6 +272,22 @@ If the `alias` property is defined for an item from `requires` section, Engine i
 
 Even if the Implementation satisfies the constraints, and the `alias` is not defined or `inject.typeInstances[].typeRef`, the TypeInstance is not injected in workflow. In this case Engine doesn't return an error.
```

Comment:
Please also add an example `interface.default` for both global and action policies inside the "Example" paragraph on https://capact.io/docs/feature/policies/global-policy and https://capact.io/docs/feature/policies/action-policy